internal/controller/stas/scan_job_controller.go

```diff
@@ -389,9 +380,24 @@ func (r *ScanJobReconciler) getScanJobLogs(ctx context.Context, job *batchv1.Job
 	return r.GetLogs(ctx, client.ObjectKeyFromObject(&jobPod), trivy.ScanJobContainerName)
 }
 
+func filterVulnerabilities(orig []stasv1alpha1.Vulnerability, minSeverity stasv1alpha1.Severity) (filtered []stasv1alpha1.Vulnerability, err error) {
```

Comment:
Not that fan of named return params. Are we using it elsewhere, or introducing it now?

Reply:
Agreed, I copied it from an example I found. I'll fix it.

Reply:
Fixed